components/forcast-summary.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import WeatherIcon from 'react-icons-weather';
+import moment from 'moment';
+
```

Comment:
Stylistic issue: most devs keep style in external style sheets. It makes applying the same style across different components much easier.